diff --git a/node_modules/@xenova/transformers/src/backends/onnx.js b/node_modules/@xenova/transformers/src/backends/onnx.js
index 0bee3dc..d435571 100644
--- a/node_modules/@xenova/transformers/src/backends/onnx.js
+++ b/node_modules/@xenova/transformers/src/backends/onnx.js
@@ -18,7 +18,9 @@
 
 // NOTE: Import order matters here. We need to import `onnxruntime-node` before `onnxruntime-web`.
 // In either case, we select the default export if it exists, otherwise we use the named export.
-import * as ONNX_NODE from 'onnxruntime-node';
+
+let ONNX_NODE = {};
+
 import * as ONNX_WEB from 'onnxruntime-web';
 
 /** @type {import('onnxruntime-web')} The ONNX runtime module. */
@@ -31,7 +33,7 @@ export const executionProviders = [
 
 if (typeof process !== 'undefined' && process?.release?.name === 'node') {
     // Running in a node-like environment.
-    ONNX = ONNX_NODE.default ?? ONNX_NODE;
+    ONNX = ONNX_NODE?.default ?? ONNX_NODE;
 
     // Add `cpu` execution provider, with higher precedence that `wasm`.
     executionProviders.unshift('cpu');
